packages/react-composites/src/composites/CallComposite/components/LocalDeviceSettings.tsx

```diff
@@ -121,52 +121,64 @@ export const LocalDeviceSettings = (props: LocalDeviceSettingsType): JSX.Element
         }}
         onRenderTitle={(props?: IDropdownOption[]) => onRenderTitle('Camera', props)}
       />
-      <Dropdown
-        label={soundLabel}
-        placeholder={defaultPlaceHolder}
-        styles={dropDownStyles(theme)}
-        disabled={!props.microphonePermissionGranted}
-        errorMessage={
-          props.microphonePermissionGranted === undefined || props.microphonePermissionGranted
-            ? undefined
-            : locale.strings.call.microphonePermissionDenied
-        }
-        options={
-          props.microphonePermissionGranted
-            ? getDropDownList(props.microphones)
-            : [{ key: 'deniedOrUnknown', text: '' }]
-        }
-        defaultSelectedKey={
-          props.microphonePermissionGranted
-            ? props.selectedMicrophone
-              ? props.selectedMicrophone.id
-              : defaultDeviceId(props.microphones)
-            : 'deniedOrUnknown'
-        }
-        onChange={(
-          event: React.FormEvent<HTMLDivElement>,
-          option?: IDropdownOption | undefined,
-          index?: number | undefined
-        ) => {
-          props.onSelectMicrophone(props.microphones[index ?? 0]);
-        }}
-        onRenderTitle={(props?: IDropdownOption[]) => onRenderTitle('Microphone', props)}
-      />
-      <Dropdown
-        placeholder={defaultPlaceHolder}
-        styles={dropDownStyles(theme)}
-        disabled={props.speakers.length === 0}
-        options={getDropDownList(props.speakers)}
-        defaultSelectedKey={props.selectedSpeaker ? props.selectedSpeaker.id : defaultDeviceId(props.speakers)}
-        onChange={(
-          event: React.FormEvent<HTMLDivElement>,
-          option?: IDropdownOption | undefined,
-          index?: number | undefined
-        ) => {
-          props.onSelectSpeaker(props.speakers[index ?? 0]);
-        }}
-        onRenderTitle={(props?: IDropdownOption[]) => onRenderTitle('Speaker', props)}
-      />
+      <Stack>
+        <Label
+          id={'call-composite-local-sound-settings-label'}
+          className={mergeStyles(dropDownStyles(theme).label)}
+          disabled={!props.microphonePermissionGranted || props.speakers.length === 0} // depends on dropdowns disabled state in the sound settings stack
```

Comment:
Shouldn't this be `&&`.

The label should be disabled if *both* `DropDown` are disabled.

Reply:
why? you can have mic but not speakers, of vise-versa.
And the only condition for "Start" button is if mic permission is granted, so I should not even test the speakers if I really want to follow the "Start" button behaviour